History.markdown

```diff
@@ -1,13 +1,20 @@
 ## HEAD
 
+## 2.4.0 / 2017-12-04
+
+### Minor
+
+  * Add meta generator (#236)
+  * Strip null values from JSON-LD hash (#249)
+
```

Comment:
I believe we also closed all self-closing tags in the template.

Reply:
#246 Was merged without @Jekyllbot

Reply:
👍 Added to the History